File: 프로그래머스/0/120906. 자릿수 더하기/자릿수 더하기.java
class Solution {
    public int solution(int n) {
        int answer = 0;

        // 정수 n의 각 자리 숫자의 합 계산
        while (n > 0) {
            answer += n % 10; // 현재 자릿수의 숫자를 더합니다.
            n /= 10; // 다음 자릿수로 이동합니다.
        }

        return answer;
    }
}

File: 프로그래머스/0/120906. 자릿수 더하기/자릿수 더하기.py
def solution(n):
    new_sum = 0
    for digit in str(n):  # `n`을 문자열로 변환하여 각 문자로 반복
        new_sum += int(digit)  # 각 문자를 정수로 변환하여 합산
    return new_sum
